### PR TITLE
99 create integration tests for database reading and writing

### PR DESCRIPTION
**Pull request by Kadi-7**

### CODE REVIEW

File: .github/workflows/db_test.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Set PYTHONPATH
+        run: echo "PYTHONPATH=$(pwd):$(pwd)/amos2023ss03-qachat" >> $GITHUB_ENV
+
+      - name: Run DB Test
+        env:
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL}}
+          SUPABASE_SERVICE_KEY: ${{ secrets.SUPABASE_SERVICE_KEY}}
+
+        working-directory: ./Testing
+        run: python db_intergration_test.py
```
Comment (emuguy1) [inline .github/workflows/db_test.yaml:32]: As we already have a workflow file, why don't you combine it? As we there already set up Python and installed the dependencies.  So no need to do this twice and increase the runtime of the CI.

Comment (Kadi-7) [reply .github/workflows/db_test.yaml:32]: I thought about separation of concerns. but actually you are right. I'm going to join them together

**Comment by emuguy1** [comment]
Great work!

**Comment by Felix-012** [comment]
looks good. though I'm a bit confused, since I thought we discussed that we either create a new class for db interaction and test those or just test our current functions interacting with the db. This mainly looks like a test class for supabase functions as far as I can see, right? (Not really a problem, just not what I expected)

**Comment by Kadi-7** [comment]
> 

yes. you are right. First thing is we use the supabase from langchain and not the original supabase lib. And in the meeting with Amela we were not able to share same idea for the necessary of db interaction class

**Review by Felix-012 — APPROVED** [review]

**Comment by amela16** [comment]
itegration tests for database

**Pull request merged**